### DISCUSSION TITLE
Does the compat package support declaritively nested routes?

### DISCUSSION THREAD

**Discussion by ripdog** [remix-run/react-router#8795] [v5 to v6 Migration]
Hi there,

For me, the biggest feature of v6 is the ability to nest routes in a single file, as opposed to spread out in different components. I've tried to do this with the compat package, but I'm getting a blank screen and no errors when I navigate to `/store-owner`:

```javascript
<Switch>
  <CompatRoute path={"/store-owner"} element={StoreOwnerRoot}>
	  <CompatRoute path={"orders/:id"} exact>
		  <StoreOwnerOrderDetail/>
	  </CompatRoute>
	  <CompatRoute path={"login"} exact>
		  <StoreOwnerLogin/>
	  </CompatRoute>
	  <CompatRoute path={'orders'} index>
		  <StoreOwnerOrders/>
	  </CompatRoute>
  </CompatRoute>
</Switch>
```

`StoreOwnerRoot` does have an `Outlet` element.

**Comment by dtsao** [comment]
CompatRoute uses the component prop. RR V6 Route has the element prop.  Change to:

<CompatRoute path={"/store-owner"} **component**={StoreOwnerRoot}>
